egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -489,6 +519,7 @@ def Main():
                 nonlin_type = args.nonlin_type,
                 nonlin_input_dim = args.nonlin_input_dim,
                 nonlin_output_dim = args.nonlin_output_dim,
+                nonlin_output_dim_init = args.nonlin_output_dim_init,
```

Comment:
Same variables should not mean different things in different contexts. Please have a separate nonlin_output_dim_{init,final} variables.